website/pages/en/index.js

```diff
@@ -0,0 +1,243 @@
+const GridBlock = CompLibrary.GridBlock;
+
+class HomeSplash extends React.Component {
+  render() {
+    const { siteConfig, language = '' } = this.props;
+    const { baseUrl, docsUrl } = siteConfig;
+    const docsPart = `${docsUrl ? `${docsUrl}/` : ''}`;
+    const langPart = `${language ? `${language}/` : ''}`;
+    const docUrl = doc => `${baseUrl}${docsPart}${langPart}${doc}`;
+
+    const SplashContainer = props => (
+      <div className="homeContainer">
+        <div className="homeSplashFade">
+          <div className="wrapper homeWrapper">{props.children}</div>
+        </div>
+      </div>
+    );
+
+    const Logo = props => (
+      <div className="mainLogo">
+        <img src={props.img_src} alt="Project Logo" />
+      </div>
+    );
+
+    const ProjectTitle = () => (
+      <h2 className="projectTitle">
+        <small>{siteConfig.tagline}</small>
+      </h2>
+    );
+
+    const PromoSection = props => (
+      <div className="section promoSection">
+        <div className="promoRow">
+          <div className="pluginRowBlock">{props.children}</div>
+        </div>
+      </div>
+    );
+
+    const Button = props => (
+      <div className="pluginWrapper buttonWrapper">
+        <a className="button" href={props.href} target={props.target}>
+          {props.children}
+        </a>
+      </div>
+    );
+
+    return (
+      <SplashContainer>
+        <Logo img_src={`${baseUrl}img/offix-background.png`} />
+        <div className="inner">
+          <ProjectTitle siteConfig={siteConfig} />
+          <PromoSection>
+            <Button href={docUrl('gettingStarted.html')}>
+              View Documentation
+            </Button>
+            <Button href="https://github.com/aerogear/offix">Github</Button>
+          </PromoSection>
+        </div>
+      </SplashContainer>
+    );
+  }
+}
+
+class Index extends React.Component {
+  render() {
+    const { config: siteConfig, language = '' } = this.props;
+    const { baseUrl } = siteConfig;
+
+    const Block = props => (
+      <Container
+        padding={['bottom', 'top']}
+        id={props.id}
+        background={props.background}
+      >
+        <GridBlock
+          align="center"
+          contents={props.children}
+          layout={props.layout}
+        />
+      </Container>
+    );
+
+    const FeatureCallout = () => (
+      <div
+        className="productShowcaseSection paddingBottom"
+        style={{ textAlign: 'center' }}
+      >
+        <h2>Introduction</h2>
+        <div>
+          Offix enables working with localy cached GraphQL compilant data
+          without access to server. It will leverage Apollo GraphQL cache do
+          deliver seamless experience for querying data even when server side
+          data is not reachable. For more information about the Apollo and
+          GraphQL please go to{' '}
+          <a href="https://www.apollographql.com/">
+            https://www.apollographql.com/
+          </a>
```

Comment:
There are quite a few typos and errors in here.

```suggestion
          Offix enables working with locally cached GraphQL compliant data
          without access to server. It leverages the <a href="https://www.apollographql.com/">Apollo GraphQL</a> cache to
          deliver a seamless experience for querying data even when server side
          data is not reachable.
```